cdap-watchdog/src/main/java/co/cask/cdap/logging/read/AvroFileReader.java

```diff
@@ -181,9 +174,34 @@ public void readLog(Location file, Filter logFilter, long fromTimeMs, long toTim
     }
   }
 
+  private List<LogEvent> readToNextSync(Filter logFilter, long fromTimeMs, DataFileReader<GenericRecord> dataFileReader,
```

Comment:
This method is reading to previous sync, right? Calling it `readToPrevSync()` might be better.